readthedocs/settings/base.py

```diff
@@ -755,11 +755,6 @@ def DOCKER_LIMITS(self):
     # Stripe
     # Existing values we use
     STRIPE_SECRET = None
-    # NOTE: Remove this when enabling djstripe,
-    # since isn't recommended to set this value,
-    # we should always use the default version
-    # that djstripe works with.
```

Comment:
Maybe we need to keep a similar comment here. Otherwise, I think we will eventually re-define the Stripe version forgetting that we shouldn't when using djstripe.

Extra points if the comment contains the variable name (`STRIPE_VERSION`) so it's easy to grep, and also a link to the djstripe documentation where it talks about this.

Reply:
there is already a comment about this in our requirements https://github.com/readthedocs/readthedocs.org/blob/f87e99f46662bd8bf0613a99814899c7066a79ee/requirements/pip.txt#L68-L72

Reply:
@stsewd doesn't that need an ignore for our automated update tooling, otherwise it will just get updated anyway?

Reply:
I was hoping the comment would call our attention if the requirement is updated, if we ignore it, we will probably never update it.

Also, the pyup bot is broken anyway... https://github.com/readthedocs/readthedocs.org/issues/9273